func_importvalue.go

```diff
@@ -0,0 +1,43 @@
+package cloudformation
+
+// ImportValue returns a new instance of ImportValue that imports valueToImport.
+func ImportValue(valueToImport string) ImportValueFunc {
```

Comment:
should valueToImport be a StringExpr or a string?  i.e. is `{"Fn::ImportValue": {"Ref": "foo"}}` a legal thing?